konfera/register/views.py

```diff
@@ -44,7 +44,7 @@ def _register_ticket_attendee(request, slug, attendee_type):
     event = get_object_or_404(Event, slug=slug)
     ticket_types = TicketType.objects.filter(event=event, attendee_type=attendee_type)
 
-    if ticket_types and len(ticket_types) == 1 and ticket_types[0].status == TicketType.ACTIVE:
+    if ticket_types and len(ticket_types) == 1 and ticket_types[0].status == TicketType.STATUSES[TicketType.ACTIVE]:
```

Comment:
`.status` returns a human-readable text see https://github.com/pyconsk/django-konfera/blob/master/konfera/models/ticket_type.py#L76 . So that's why it evaluated to `False`.